packages/aws-lambda/layer/bin/publish-layer.sh

```diff
@@ -193,30 +211,36 @@ if [[ $BUILD_LAYER_WITH == local ]]; then
 
   echo "Building local tar.gz for @instana/core."
   cd core
+  rm -rf instana-core-*.tgz
   npm --loglevel=warn pack
   mv instana-core-*.tgz $LAYER_WORKDIR/instana-core.tgz
 
   echo "Building local tar.gz for @instana/serverless."
   cd ../serverless
+  rm -rf instana-serverless-*.tgz
```

Comment:
When you previously ran the script it will happen that you get two zips in the folder e.g. 
instana-serverless-2.6.2
instana-serverless-2.6.3

> mv instana-serverless-*.tgz $LAYER_WORKDIR/instana-serverless.tgz

Will then fail